citizenvoice/apiapp/templates/survey/base.html

```diff
@@ -0,0 +1,17 @@
+{% load static %}
```

Comment:
Are the HTML templates necessary? The API doesn't need to render data in HTML. What is the reason behind this?

Reply:
This was only for debugging for myself, I completely forgot to delete them!